packages/assets-controllers/src/TokenBalancesController.ts

```diff
@@ -116,7 +116,7 @@ export class TokenBalancesController extends BaseController<
     }
     const { tokens } = this.config;
     const newContractBalances: { [address: string]: BN } = {};
-    for (const i in tokens) {
+    for (const [i] of tokens.entries()) {
```

Comment:
To access array indices in a `for..of` loop, use the new `Array.prototype.entries()` method.